### PR TITLE
Add LICENSE to project

### PR DESCRIPTION
**Pull request by kshithijiyer**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```
Comment (amarts) [inline LICENSE]: Can we please use LGPLv3 ?  (or Apache v2.0) is also fine.

As this is not directly linking the code of glusterfs, it doesn't bind to have a GPLv2 licence. Lets make it more adaptive in other echo systems, and keep Apache v2.0 IMO.

Comment (kshithijiyer) [reply LICENSE]: LGPLv3 looks good to me @amarts, however moving to Apache v2.0 looks a little tricky as the project is originally open sourced under LGPLv2 moving on would require us to communicate the same to all contributors

Comment (kshithijiyer) [reply LICENSE]: One such example is: https://github.com/gluster/glustercli-python/issues/28

Comment (amarts) [reply LICENSE]: Ack, lets go with LGPLv3, which is compatible with Apachev2.0

Comment (kshithijiyer) [reply LICENSE]: Done @amarts

**Review by amarts — APPROVED** [review]

**Comment by kshithijiyer** [comment]
@aravindavk Can you please review?

**Review by aravindavk — APPROVED** [review]

**Comment by kshithijiyer** [comment]
@aravindavk @amarts Can we merge this patch?

**Pull request merged**